apollo-router/src/configuration/mod.rs

```diff
@@ -418,13 +426,13 @@ mod tests {
                     (
                         "inventory".to_string(),
                         Subgraph {
-                            routing_url: "http://inventory/graphql".to_string(),
+                            routing_url: Url::parse("http://inventory/graphql").expect("test"),
```

Comment:
:yarn: My rule of thumb is:
- always use `expect("write some proof; qed")` if not in a test
- use `unwrap()` if very very obvious (like getting the first item of a [Split iterator](https://doc.rust-lang.org/std/primitive.str.html#method.split))
- use always `unwrap()` in tests unless you want to provide a better error message to the developer

Here clearly "test" won't help much the developer xD so you can just `unwrap()` things.

```suggestion
                            routing_url: Url::parse("http://inventory/graphql").unwrap(),
```

Reply:
oh, I see the logic now, thanks :)